frontend/src/util/coordinateConversion.ts

```diff
@@ -0,0 +1,61 @@
+// get decimal part by converting to string to avoid false precision
+const getDecimalPart = (value: number) => {
+  const stringValue = String(value)
+  const index = stringValue.indexOf('.')
+  return Number('0.' + (index > -1 ? stringValue.substring(index + 1) : '0'))
+}
+
+export const convertDecToDms = (dec: number | null | undefined, isLatitude: boolean) => {
```

Comment:
Could it be clearer for `isLatitude` to be a string type that accepts `"latitude"` or `"longitude"`? It is a bit unclear what the boolean means when calling the function: `convertDecToDms(<some number>, true)`.

Reply:
true, will change